src/lib.rs.in

```diff
@@ -613,6 +642,9 @@ class IronSdk {
     ///
     /// @return details about the newly created device
     static_method generate_new_device(_:&str, _:&str, _: &DeviceCreateOpts) -> Result<DeviceContext, String>; alias generateNewDevice;
+
+    method IronSdkAdvanced::advanced(&self) -> IronSdkAdvanced<'a>;
```

Comment:
We should probably put a javadoc comment here.

Reply:
Done